spec/models/dynamic_field_group_spec.rb

```diff
@@ -160,14 +160,7 @@
     end
 
     context 'when creating a group that shares the same string_key as a non-sibling group' do
-      let(:dynamic_field_group) { FactoryBot.create(:dynamic_field_group) }
-      let(:dynamic_field) { FactoryBot.build(:dynamic_field, string_key: 'name', dynamic_field_group: dynamic_field_group) }
```

Comment:
I think this was copied from another test.  It didn't seem to be testing anything relevant to this group, so that's why I removed it.  (We can re-add if I'm mistaken!)